service/rpc1/client.go

```diff
@@ -23,10 +22,10 @@ type RPCClient struct {
 var errAPIUnsupported = errors.New("unsupported")
 
 // NewClient creates a new RPCClient.
-func NewClient(addr string) *RPCClient {
+func NewClient(t testing.TB, addr string) *RPCClient {
```

Comment:
Why not do the same thing to service/rpc2.NewClient as well? They could be renamed to NewTestClient.

Reply:
Agree, thanks.
Renamed.